### PR TITLE
Add kickstart files for RHEL7 STIG and STIG with GUI.

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Add kickstart files for RHEL7 STIG and STIG with GUI.

### CODE REVIEW

File: rhel7/kickstart/ssg-rhel7-stig_gui-ks.cfg [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+logvol /var/log/audit --fstype=xfs --name=LogVol05 --vgname=VolGroup --size=512 --fsoptions="nodev"
+logvol swap --name=lv_swap --vgname=VolGroup --size=2016
+
+%addon org_fedora_oscap
+        content-type = scap-security-guide
+        profile = xccdf_org.ssgproject.content_profile_stig_gui
+%end
+
+# Packages selection (%packages section is required)
+%packages
```
Comment (mildas) [inline rhel7/kickstart/ssg-rhel7-stig_gui-ks.cfg:118]: @ggbecker What do you think about installing GUI packages by default? Adding `@^graphical-server-environment` package should do it.

Comment (mildas) [reply rhel7/kickstart/ssg-rhel7-stig_gui-ks.cfg:118]: However, I don't know how Anaconda installation will behave - e.g. if user selects `stig_gui` profile and `Server` package selection.

Comment (ggbecker) [reply rhel7/kickstart/ssg-rhel7-stig_gui-ks.cfg:118]: I've added the package group: `@^graphical-server-environment` to the list of packages to be installed.

**Comment by mildas** [comment]
Tested and works fine. Thank you

**Pull request merged**